instrumentation/events/lttng-module/net.h

```diff
@@ -258,6 +412,14 @@ LTTNG_TRACEPOINT_EVENT_CLASS(net_dev_template,
 					/* For any other header type, there is nothing to do. */
 					break;
 				}
+
+				/* Copy the transport header */
+				if (__get_transport_header_type(skb) == TH_TCP) {
+					ctf_array_type(uint8_t, tcp_hdr(skb),
+							sizeof(struct tcphdr))
+					ctf_align(uint8_t)
```

Comment:
Or rather should I just 8-bits align everything to avoid any misalignment at read time?